pyscriptjs/tests/unit/pyodide.test.ts

```diff
@@ -2,6 +2,7 @@ import type { AppConfig } from '../../src/pyconfig';
 import { Runtime } from '../../src/runtime';
 import { PyodideRuntime } from '../../src/pyodide';
 import { CaptureStdio } from '../../src/stdio';
+import { make_PyScript } from '../../src/components/pyscript';
```

Comment:
Why is this imported here?

Reply:
Oops this was a mistake (I thought I was poking at it on Jeff's branch but seems not 😬 )